grib/build.gradle

```diff
@@ -17,7 +17,7 @@ dependencies {
     compile libraries["guava"]
     compile libraries["jcommander"]
     compile libraries["slf4j-api"]
-    testCompile libraries["truth"]
+    compile libraries["truth"]

```

Comment:
Any reason this moved back to the `compile` scope? I don't see it used in any of these changes.